client/signup/main.jsx

```diff
@@ -779,6 +782,21 @@ class Signup extends Component {
 		}
 	}
 
+	shouldShowZendeskPresalesChat() {
+		const isEnglishLocale = config( 'english_locales' ).includes( getLocaleSlug() ?? '' );
+		const currentTime = new Date();
+
+		return (
+			! this.props.isLoggedIn &&
+			( isCurrentPlanFlow( this.props.flowName ) || this.props.flowName === 'onboarding' ) &&
+			isEnglishLocale &&
+			currentTime.getUTCHours() >= 15 &&
+			currentTime.getUTCHours() < 21 &&
+			currentTime.getUTCDay() !== 0 &&
+			currentTime.getUTCDay() !== 6
```

Comment:
That might be worth putting into a configuration section somewhere, with some comments, to make it easier in case we decide to tweak it in the future.